analyzer/src/main/resources/scripts/bundler_dependencies_metadata.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/ruby
+
+# Copyright (C) 2020 Bosch.IO GmbH
+# Copyright (C) 2017-2019 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+#
+# This script mimics the behavior of calling the CLI command `bundle lock`, which
```

Comment:
I think you should separate this documentation from the license with an empty line